lib/covered/files.rb

```diff
@@ -134,7 +134,7 @@ def initialize(output, path)
 		attr :path
 		
 		def expand_path(path)
-			File.expand_path(super, @path)
```

Comment:
I think this is actually correct - it's expanding `super(path)` relative to `@path`.